### PR TITLE
fix: add app store links to sharing page

### PR DESCRIPTION
**Pull request by Sherange**
## 🐛 Issue

Add app store links to sharing page
Closes https://github.com/ApollosProject/apollos-embeds/issues/214

## ✏️ Solution

add app store links to sharing page

## 🔬 To Test

1. Web shared link -> check page footer
2. http://localhost:3000/?root=FeatureFeed-cc9c554e-7e37-450b-8707-2bb0434ebca6

## 📸 Screenshots

Screenshot 2024-06-07 at 3 04 46 PM

### CODE REVIEW

**Comment by Sherange** [comment]
@redreceipt Can I have your comments on UI  (store image size, spacing and font size)

File: micro-service/src/App.js [hunk 1]
```diff
@@ -25,6 +26,9 @@ Sentry.init({
   tracesSampleRate: 1.0,
 });
 
+const appStoreUrl = process.env.NEXT_PUBLIC_APPLE_APP_STORE_LINK;
```
Comment (redreceipt) [inline micro-service/src/App.js:29]: These values should come from the database configs

**Review by redreceipt — CHANGES_REQUESTED** [review]

File: micro-service/src/App.js (hunk 1 above)
Comment (Sherange) [reply micro-service/src/App.js:29]: @redreceipt  do we config this from admin I cant find the config key of store url in database

Comment (redreceipt) [reply micro-service/src/App.js:29]: https://github.com/ApollosProject/apollos-cluster/blob/fa74dd419a0fc527832c7b3b38066f52ab9259f3/src/data/sharable/index.js#L30-L31

**Comment by Sherange** [comment]
@redreceipt if one of the store links not configured do you want to hide that store image ?

**Comment by redreceipt** [comment]
> @redreceipt if one of the store links not configured do you want to hide that store image ?

let's not show them unless we have both

**Review by redreceipt — APPROVED** [review]

**Pull request merged**